cps/kni.c

```diff
@@ -979,6 +982,69 @@ moderror(int err)
 	}
 }
 
+#define SYS_MODULES_ATTR_PATH ("/sys/module/%s/%s")
+
+static int
+get_loaded_kmod_attr(const char *attr, char *val, size_t val_len)
+{
+	FILE *attr_file;
+	char path[128];
+	char line[64];
+	int ret;
+
+	ret = snprintf(path, sizeof(path), SYS_MODULES_ATTR_PATH,
+		KNI_MODULE_NAME, attr);
+	RTE_VERIFY(ret > 0 && ret < (int)sizeof(path));
```

Comment:
Given that `attr` is a parameter, turn this `RTE_VERIFY()` into an `if` statement and return an error if needed.